wandb/proto/wandb_settings.proto

```diff
@@ -23,6 +23,12 @@ message OpenMetricsFilters {
   }
 }
 
+message RunMoment {
+  google.protobuf.StringValue run = 1;
+  google.protobuf.DoubleValue value = 2;
+  google.protobuf.StringValue metric = 3;
```

Comment:
You can just do this:

```protobuf
string run = 1;
double value = 2;
string metric = 3;
```

If you need to distinguish between unset fields and zero values, use `optional`

```protobuf
optional string run = 1;
optional double value = 2;
optional string metric = 3;
```

but in that case, document what the difference is.

Reply:
done